packages/app-project/pages/projects/[owner]/[project]/about/faq.js

```diff
@@ -0,0 +1,12 @@
+import getDefaultPageProps from "@helpers/getDefaultPageProps"
+export { default } from "@screens/ProjectAboutPage"
+
+export async function getServerSideProps({ params, query, req, res }) {
+  const { props } = await getDefaultPageProps({ params, query, req, res })
```

Comment:
Could the About pages use `getStaticProps`, and `getStaticPaths`, so that we can serve them via our CDN?

Reply:
Agreed! I think you're right the page data could be cached - I'll try those functions 😃